CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Analysis/CaptureSuggestionsView.swift

```diff
@@ -14,63 +14,88 @@ final class CaptureSuggestionsView: UIView {
     fileprivate let suggestionContainer: UIView
-    fileprivate let suggestionTitle: UILabel
-    fileprivate let containerHeight: CGFloat = 135
-    fileprivate let suggestionTitleHeight: CGFloat = 20
+    fileprivate let containerHeight: CGFloat = 96
     fileprivate var itemSeparationConstraint: NSLayoutConstraint = NSLayoutConstraint()
     fileprivate var bottomConstraint: NSLayoutConstraint = NSLayoutConstraint()
     fileprivate let repeatInterval: TimeInterval = 5
     fileprivate let superViewBottomAnchor: NSLayoutYAxisAnchor
-    fileprivate let suggestionIconImage = UIImage(named: "analysisSuggestionsIcon",
-                                                  in: giniCaptureBundle(), compatibleWith: nil)
-    fileprivate var suggestionTexts: [String] = [
+
+    fileprivate var suggestionIconImages = [
+        UIImageNamedPreferred(named: "captureSuggestion1"),
+        UIImageNamedPreferred(named: "captureSuggestion2"),
+        UIImageNamedPreferred(named: "captureSuggestion3"),
+        UIImageNamedPreferred(named: "captureSuggestion4")
+    ]
+
+    fileprivate var suggestionTitle: [String] = [
         .localized(resource: AnalysisStrings.suggestion1Text),
         .localized(resource: AnalysisStrings.suggestion2Text),
         .localized(resource: AnalysisStrings.suggestion3Text),
         .localized(resource: AnalysisStrings.suggestion4Text)
     ]
+
+    fileprivate var suggestionDescription: [String] = [
+        .localized(resource: AnalysisStrings.suggestion1Details),
+        .localized(resource: AnalysisStrings.suggestion2Details),
+        .localized(resource: AnalysisStrings.suggestion3Details),
+        .localized(resource: AnalysisStrings.suggestion4Details)
+    ]
     
-    init(superView: UIView, bottomAnchor: NSLayoutYAxisAnchor, font: UIFont) {
+    init(superView: UIView, bottomAnchor: NSLayoutYAxisAnchor, font: GiniCaptureFont, multiPageEnabled: Bool) {
+        if multiPageEnabled {
+            suggestionIconImages.append(UIImageNamedPreferred(named: "captureSuggestion5"))
+            suggestionTitle.append(.localized(resource: AnalysisStrings.suggestion5Text))
+            suggestionDescription.append(.localized(resource: AnalysisStrings.suggestion5Details))
+        }
+
         suggestionContainer = UIView()
-        suggestionTitle = UILabel()
-        suggestionText = UILabel()
+        if #available(iOS 13.0, *) {
+            suggestionContainer.backgroundColor = Colors.Gini.dynamicPearl
+        } else {
+            suggestionContainer.backgroundColor = Colors.Gini.pearl
+        }
+        suggestionContainer.layer.cornerRadius = 16
+        
+        suggestionTextTitle = UILabel()
         superViewBottomAnchor = bottomAnchor
 
-        suggestionIcon = UIImageView(image: suggestionIconImage)
+        suggestionTextTitle.textColor = UIColor.from(giniColor: GiniColor(lightModeColor: .black, darkModeColor: .white))
+        suggestionTextTitle.font = font.with(weight: .bold, size: 16, style: .body)
```

Comment:
Please, use font in such way:
        cell.headerLabel.font = giniConfiguration.textStyleFonts[.headline]